### ISSUE TITLE
Decomposition of volume into ellipsoids

### ISSUE THREAD

**Issue by alpha027** [kmammou/v-hacd#97]
Hi, 

I want to know if the decomposition of a mesh (closed volume) into ellipsoids is supported in this repository.
If the decomposition is available could you make reference to it in code please ?

Thanks.

**Comment by jratcliff63367** [comment]
No, all you can do is approximate convex hulls as elipsoids which will be a pretty ugly fit in most cases.
